packages/cms/express-mini-apps/two-factor-auth/route.ts

```diff
@@ -204,6 +204,81 @@ export function twoFactorAuthRoute(
       }
     })
 
+    // clear exist secret
+    app.post('/api/2fa/clear', async (req, res) => {
+      const token = req.body?.token
+      if (!token) {
+        res.status(422).send({ status: 'error', message: 'no token' })
+        return
+      }
+
+      const context = await commonContext.withRequest(req, res)
+      const currentSession = context?.session
+      if (!currentSession) {
+        res.status(403).send({ status: 'fail', message: 'no session' })
```

Comment:
question:
`401` status code 是否比較適合？

suggestion:
In JSend format, `fail` status response should have `data` property, rather than `message` property.

Reply:
done